test/discharge-wizard/00-required.e2e.spec.js

```diff
@@ -42,12 +42,9 @@ module.exports = E2eHelpers.createE2eTest(
       .axeCheck('.main')
       .click('a.usa-button-primary');
 
-    client.pause(Timeouts.verySlow);
-
     // results page
     client
-      .click('a.usa-button-primary')
-      .waitForElementVisible('.dw-guidance', Timeouts.verySlow)
+      .waitForElementVisible('.dw-guidance', Timeouts.slow)
```

Comment:
It seems like this removes a button click. Is that intentional?

Reply:
Yeah it does. That click was unintentionally in there twice and the second click was causing an error.